apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -98,7 +131,24 @@ export default function LandingPage({
       <HeaderBannerNoImage
         headingText={i18n.professionalLearning()}
         backgroundColor={color.light_gray_50}
-      />
+      >
+        {availableTabs && (
```

Comment:
I think if we only have one tab, we shouldn't show any tabs, so we might want this to be `availableTabs > 1`. (`currentTab` in that case could default to the first of `availableTabs` and never be changeable.)